16/alpine3.11/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM alpine:3.11
 
-ENV NODE_VERSION 16.3.0
+ENV NODE_VERSION 16.4.0
```

Comment:
this change should not be included in this PR

Reply:
Is there a way to re-generate those files from the template without updating the version? I tested `./update.sh -b`, but that just fails.

Reply:
I'd just revert the changes manually, I don't think our script supports "generate from template, but keep current version"